packages/ref-data-uploaders/txc-uploader/txc_processor.py

```diff
@@ -541,6 +593,18 @@ def write_to_database(
                                 logger
                             )
 
+                for vehicle in vehicle_journeys:
```

Comment:
We are running insert query within a for loop. Could we run a batch insert to insert many vehicle journey rows?

Reply:
just following how journey patterns does it

Reply:
Journey pattern sections depends on the journey pattern id - in this case there are no dependency that would require individual insert

Reply:
done